docs/native/src/faq/filetypes.rst

```diff
@@ -0,0 +1,16 @@
+What file types are supported in |Office iOS|?
```

Comment:
I've reached out to the UA team to write a new PowerPoint compatible files article, but I think we should plan to publish this page in the meantime. As Karvell mentioned, we get asked for this information a lot.